source/code/plugins/oms_common.rb

```diff
@@ -17,6 +17,7 @@ class Common
     require 'digest'
     require 'date'
     require 'securerandom'
+    require 'resolv-replace'
```

Comment:
we don't need any additional gem for this ?

Reply:
No, it's in the Ruby stdlib.